terraform/modules/serverless-api-dns-for-failover/main.tf

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
You can use `locals.aws_region` and `locals.aws_region.secondary` instead of declaring variables. Eliminates the possibility of the provider and variables getting out of sync. (The same can be done in the serverless-api-dns-for-failover module.)

https://registry.terraform.io/providers/hashicorp/aws/4.8.0/docs/data-sources/region

```suggestion
locals {
  aws_region           = data.aws_region.primary.name
  aws_region_secondary = data.aws_region.secondary.name
}

data "aws_region" "primary" {}
data "aws_region" "secondary" {
  provider = aws.secondary
}

```

Reply:
Great suggestion. Thanks!